_commands/toys/cowsay.md

```diff
@@ -2,23 +2,111 @@
 ---
 
 cowsay
--------
+--
+`cowsay` generates an ASCII picture of a cow saying something provided by  the  user.
 
-ToDo: Needs Completion
 
 ~~~ bash
-*short* example of command usage and output
+$ cowsay Hi There
+ __________ 
+< Hi There >
+ ---------- 
+        \   ^__^
+         \  (oo)\_______
+            (__)\       )\/\
+                ||----w |
+                ||     ||
 ~~~
 
 <!--more-->
 
+
+
 ### Useful Options / Examples
+There are many useful options for this intersting cowsay command.
+The complete list is: cowsay  `[-e  eye_string]` `[-f cowfile]` `[-h]` `[-l]` `[-n]` `[-T tongue_string]` `[-W column]` `[-bdgpstwy]`
+
+`-e` allows the user to customize the cow's eyes which are defaulted to "oo" as seen above.
+`-T` allows the user to add a tongue string to the cow image. By default there is no tongue string.
+`-W` allows the user to customize the number of character printed in the message before the message wraps around
+`-[-bdgpstwy]` are all options to define different pre-made cows emotional states. For example, adding the -g flag will result in a greedy cow to print and -s will result in a stoned cow to print.
 
-#### Example command
+#### cowsay [-bdgpstwy] [`user_input`]
+
+~~~ bash
+$ cowsay -s hi there
+ __________ 
+< hi there >
+ ---------- 
+        \   ^__^
+         \  (**)\_______
+            (__)\       )\/\
+             U  ||----w |
+                ||     ||
+~~~
 
 ##### Break it down
+There are several provided modes which change the appearance of the cow depending on its particular emotional/physical state
+The user can set this state using these flags
+####cowsay [-f cowfile] [`user_string`]
```

Comment:
its all good. Nit picking can be helpful. So i should only use backticks on flags and options. Do i only use these backticks when explaining the options? in other words, I wouldn’t use them in the ~~~bash sections, correct?
furthermore, on one of your previous emails you mentioned that a newline would help. Are you saying I should add a newline after the line “The user can set this stat using these flags”
let me know.

thanks

> On Mar 22, 2016, at 10:44 PM, Alex Chojnacki <notifications@github.com> wrote:
> 
> In _commands/toys/cowsay.md <https://github.com/c4cs/c4cs.github.io/pull/96#discussion_r57102946>:
> 
> >  
> >  ##### Break it down
> > +There are several provided modes which change the appearance of the cow depending on its particular emotional/physical state
> > +The user can set this state using these flags
> > +####cowsay [-f cowfile] [`user_string`]
> Also, just for maximal clarity (please don't hate me for the nitpicking) we put backticks around all the flags to indicate that they are meant for the command line. Basically anything that you would type as part of the command, its structure, or its output, we put it in backticks to indicate that it is related to the command line.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/c4cs/c4cs.github.io/pull/96/files/c8fe36e731a610e6cff724ad7fd6a3977c73b117#r57102946>